### PR TITLE
PLAT-122723: Fixed 5-way navigation between Scrollbar Buttons in VirtualList

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed

* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
- when focusableScrollbar was true for VirtualList and Spotlight is on the `Up` ScrollButton in the VirtualList,  press 5-way Down
button did not move  Spotlight on the `Down` ScrollButton, but on an Item

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- added state for focusableScrollButtons in useSCroll/scrollButtons.js

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
- No changelog because a private component was modified

### Links
[//]: # (Related issues, references)
PLAT-122723

### Comments

Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### CODE REVIEW

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Comment by jeonghee27** [comment]
>No changelog because a private component was modified

Please write chagelog since a bug has been fixed.

**Comment by daniel-stoian-lgp** [comment]
> > No changelog because a private component was modified
> 
> Please write chagelog since a bug has been fixed.

Ok, I added it

File: useScroll/ScrollButtons.js [hunk 1]
```diff
@@ -102,7 +104,7 @@ const useScrollButtons = (props) => {
 
 	function onKeyDownButton (ev, position) {
 		const
-			{focusableScrollButtons, vertical, preventBubblingOnKeyDown} = props,
+			{vertical, preventBubblingOnKeyDown} = props,
```
Comment (jeonghee27) [inline useScroll/ScrollButtons.js:107]: Wow, this issue is pretty difficult .
onKeyDown has closure value from the previous prop, and it seems that the closure value is not updated even if the prop is changed.
your approch is not fix the cause and just make sure to get the latest focusableScrollButtons.
Is it correct that I analyzed?
If right , can you add a more description about it?
I should discuss this with our members.

Comment (daniel-stoian-lgp) [reply useScroll/ScrollButtons.js:107]: Yes, you are correct. I didn't know any other approach to solve it

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Comment by MikyungKim** [comment]
Closing in favor of #607.

**Pull request closed without merging**